docker-compose.yml

```diff
@@ -9,6 +9,7 @@ web:
   environment:
     PG_USERNAME: postgres
     PG_HOST: db
+  command: bundle exec unicorn -l 0.0.0.0:3000
```

Comment:
You have the CMD in your Dockerfile so you don't need to specify it separately in docker-compose.yml